product_docs/docs/pgd/5/upgrades/index.mdx

```diff
@@ -189,7 +189,7 @@ that the upgraded node is working as expected.
 ## Moving from HARP to PGD-Proxy
 
 HARP can for a time coexist with the new
-[Connection management](/product_docs/docs/pgd/5/routing) configuration.
+[Connection management](../routing) configuration.
```

Comment:
@drothery-edb just wanted to confirm, is there any guidelines around how these link should be

For example, if not wrong, in this case, I guess both of these will work fine but I guess first one will create problem if we move this as it is in v6 doc?

- `[Connection management](/pgd/5/routing)`
- `[Connection management](../routing)`

Reply:
The form we can use if we want the link to go to the latest version is: [Connection management](/pgd/latest/routing)

Reply:
I personally find that format the easiest.

Reply:
Yes, I mean in this case, we don't want it to go to the latest...but with whatever version user has selected currently in browser (because they are mostly navigating from 1 page to another and mostly they want to stay on same version).

Reply:
I added the deploy label. It kicks off a draft deployment. It takes a while though. I kind test the links in my local build if you like.

Reply:
ok, thanks.